src/style.css

```diff
@@ -1 +1,69 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.logo__link {
+  margin: 10px 50px;
+}
+
+.nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  margin: 0;
+  padding-right: 40px ;
+}
+
+.nav__item {
+  display: flex;
+  position: relative;
+  /* flex-direction: column;
+  justify-content: flex-start; */
+}
+
+.nav__link {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: black;
+  padding: 23px 10px;
```

Comment:
For fix tests remove horizontal paddings.
Just set nav_item
```
.nav__item:not(:last-child) {
  margin-right: 20px
}

Reply:
But if I remove 23px this rule not work.           **Links in nav should have clickable area above and below the text**